Dockerfile

```diff
@@ -45,11 +45,11 @@ COPY Cargo.toml Cargo.lock ./
 COPY . .
 
 # Build the project in release mode for the MUSL target
-RUN cargo build --release --bin pubky-homeserver --target $TARGETARCH-unknown-linux-musl
+RUN cargo build --release --bin pubky-homeserver --bin pubky-testnet --target $TARGETARCH-unknown-linux-musl
```

Comment:
do you still need to run the homeserver? If so, I can just run one in the binary of the testnet as well as an http_relay...

Reply:
I will do you one better, I publish `pubky-testnet` crate tonight, and you remove this entire thing, and wherever you need docker just `cargo install pubky-testnet` and then run it, no need for the docker file to be here.

Reply:
No, it's for optionality. I've now made it build one or the other, defaulting to testnet.

Reply:
Just seen your second message. As long as we can retain control of compatibility, I'm open to this.

Reply:
Published `pubky-testnet` and now you can test it as I described.